src/Symfony/Component/Console/Application.php

```diff
@@ -613,6 +618,14 @@ public function find($name)
 
         // if no commands matched or we just matched namespaces
         if (empty($commands) || count(preg_grep('{^'.$expr.'$}i', $commands)) < 1) {
+            // offer list if this is just NS (e.g. "make" and not "make:v")
```

Comment:
Instead of putting this code here, we could modify the `findAlternatives()` method to return the full list of commands when a namespace is given. That way we maintain the error exit code, which is important because the command is not found.